### ISSUE TITLE
Add SAEF metadata block to Demo Dataverse for testing

### ISSUE THREAD

**Issue by jggautier** [IQSS/dataverse.harvard.edu#171]
@cmbz and @sbarbosadataverse need to test a metadata block for a collection of data they plan to add to the Harvard Dataverse Repository, and would like the metadata block to be added to the Demo Dataverse. (I created a Dataverse instance on AWS with the new metadata block, but we aren't able to save datasets and see what the search/browse experience is like.)

Could this metadata block be added to the Demo Dataverse?

A .tsv file and properties file is in [customSAEF_TSV_and_properties_files.zip](https://github.com/IQSS/dataverse.harvard.edu/files/8962263/customSAEF_TSV_and_properties_files.zip)

**Comment by qqmyers** [comment]
FWIW: That's an odd error at dataset save on your AWS instance - an 'onclick redefined' issue appearing in the UI. In general, once you add a custom block, you need to update the solr schema to add the new fields (and dataset save will fail without that), but I haven't seen the onclick error message before. So - checking the log/updating the schema if you haven't might get that AWS instance working, but minimally it might be worth checking the log to see if there is anything else going wrong (that would could see later or may show up on demo if it is really related to the new block somehow).

**Comment by jggautier** [comment]
Thanks for checking @qqmyers. Yeah I didn't do this step of [updating the solr schema](https://guides.dataverse.org/en/latest/admin/metadatacustomization.html#updating-the-solr-schema), I wasn't sure if I needed to and the instructions confuse me a bit.

I also don't know how to check the logs or what I'd be looking for. Figured asking if a developer could add the metadata block to the Demo Dataverse would be faster than someone who knows more helping me fix the AWS instance.

Although in the long term maybe it would be more efficient if I and other non-developer folks knew how to create AWS instances and add metadata blocks to test without this odd error.

**Comment by jggautier** [comment]
@scolapasta, @sbarbosadataverse asked me to ask that this be added to a spring

**Comment by scolapasta** [comment]
Added to demo. For future, AWS can be spun up with a custom block if it's set up on lines 99-102 of: https://github.com/GlobalDataverseCommunityConsortium/dataverse-ansible/blob/develop/defaults/main.yml#L99

**Comment by jggautier** [comment]
Thanks @scolapasta. I tried what you mentioned in late June and ran the issues with saving the datasets, which @qqmyers tried to help troubleshoot earlier in this GitHub issue but I don't have the technical know-how to do.

Today it worked, and the same steps I used today to create http://ec2-3-86-198-239.compute-1.amazonaws.com/dataverse/root are what I used in June. So it still seems hit or miss, maybe sometimes based on whether I use the Dataverse repo's develop branch or master branch (today I used the master branch) or maybe if the ansible GitHub repo is updated to work with the latest version of the Dataverse software.
